addons/sale/wizard/sale_make_invoice_advance_views.xml

```diff
@@ -62,7 +62,7 @@
     </record>
 
     <record id="action_view_sale_advance_payment_inv" model="ir.actions.act_window">
-        <field name="name">Create invoices</field>
+        <field name="name">Create invoice</field>
```

Comment:
Well, for me it makes sense to keep the plural since it's an action used to potentially invoice multiple SO's at once from the list view :thinking: